src/store.ts

```diff
@@ -37,9 +39,29 @@ export const initialStoreState: C4CState = {
   authenticationState: initialUserState,
 };
 
+export const LOCALSTORAGE_STATE_KEY: string = 'state';
```

Comment:
Sorry if this is a dumb question but is there the possibility of this colliding with other local storage elements the user may already have from other websites? If so we should name this something more C4C specific

Reply:
I believe local storage is scoped by domain, otherwise that would be a massive security vulnerability in every website? I'm not actually 100% sure though.

Reply:
Local storage is unique per protocol (http/https), domain, and port. So we should be good!